type_test.go

```diff
@@ -0,0 +1,105 @@
+package hubspot_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/belong-inc/go-hubspot"
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestHsStr_String(t *testing.T) {
+	tests := []struct {
+		name string
+		hs   *hubspot.HsStr
+		want string
+	}{
+		{
+			name: "Success",
+			hs:   hubspot.NewString("text"),
+			want: "text",
+		},
+		{
+			name: "Success case of nil receiver",
```

Comment:
[opt] Honestly I don't prefer test name that will be modified in the execution result.  Once it failed, it's troublesome to search the test case. snake_case or camelCase (PascalCase) would be smarter.

Reply:
I will fix it.